demos/button.html

```diff
@@ -75,6 +75,42 @@
           </div>
         </fieldset>
         <fieldset>
+          <legend>Graceful degraded buttons</legend>
+          <button class="mdc-button" no-js>
```

Comment:
can we make those attributes `data-demo-no-js` instead of `no-js` so that they are valid HTML attrs? I also think having the `demo` in there is useful since it signals to users that the data attribute is not part of the component's API and exists for demo purposes only